test/definitions/3.0.0/models/channel/channel/channel.js

```diff
@@ -0,0 +1,66 @@
+const Ajv = require('ajv');
+const assert = require('assert');
+const addFormats = require('ajv-formats');
+const fs = require('fs');
+
+const ajv = new Ajv({
+  jsonPointers: true,
+  allErrors: true,
+  schemaId: '$id',
+  logger: false,
+  validateFormats: true,
+  strict: false,
+});
+addFormats(ajv);
+
+const jsonSchemaName = 'Channel';
+const jsonSchema = require('../../../../../../definitions/3.0.0/channel.json');
+const schemesV3_0_0 = require('../../../../../ajv-schemes');
+const validator = schemesV3_0_0(ajv).compile(jsonSchema);
+
+describe.skip(`${jsonSchemaName}. Reason: multiple errors with bindings, externalDoc, ...`, () => {
+  it('example', () => {
+    const info = JSON.parse(fs.readFileSync(`${__dirname}/example.json`, 'utf-8'));
+    const validationResult = validator(info);
+
+    assert(validationResult === true, `${jsonSchemaName} example MUST be valid`);
+  });
+
+  it('empty', () => {
+    const info = JSON.parse(fs.readFileSync(`${__dirname}/empty.json`, 'utf-8'));
+    const validationResult = validator(info);
+
+    assert(validationResult === true, `${jsonSchemaName} with empty body is valid`);
+  });
+
+  it('without required properties', () => {
+    const info = JSON.parse(fs.readFileSync(`${__dirname}/without required properties.json`, 'utf-8'));
+    const validationResult = validator(info);
+
+    assert(validationResult === true, `${jsonSchemaName} without required properties is valid`);
+  });
+
+  it('only required properties', () => {
+    const info = JSON.parse(fs.readFileSync(`${__dirname}/only required properties.json`, 'utf-8'));
+    const validationResult = validator(info);
+
+    assert(validationResult === true, `${jsonSchemaName} is valid with only required properties`);
+  });
+
+  it('extended', () => {
+    const info = JSON.parse(fs.readFileSync(`${__dirname}/extended.json`, 'utf-8'));
+    const validationResult = validator(info);
+
+    assert(validationResult === true, `${jsonSchemaName} can be extended`);
+  });
+
+  it('wrongly extended', () => {
```

Comment:
Why use such a huge file for something so simple as testing `wrong` property 🤨

Reply:
You mean json file to test? It was copy paste. I'll shortener them

Reply:
Simplified